python/pyspark/sql/types.py

```diff
@@ -641,6 +642,50 @@ def __eq__(self, other):
         return type(self) == type(other)
 
 
+class PointUDT(UserDefinedType):
```

Comment:
This class is only used in tests, right?  If so, I think we should move it to a tests file.